index.bs

```diff
@@ -1141,6 +1143,7 @@ Possible values are:
 <li>"<dfn><code>source-destination-limit-replaced</code></dfn>"
 <li>"<dfn><code>source-destination-per-day-rate-limit</code></dfn>"
 <li>"<dfn><code>source-destination-rate-limit</code></dfn>"
+<li>"<dfn><code>source-max-event-states-limit</code></dfn>"
```

Comment:
This needs to be added to https://github.com/WICG/attribution-reporting-api/blob/590c74ec584a22268f6c621a1e3d162486421a04/ts/src/constants.ts#L64.